integration_test/test/code_generation/app_with_mssql_adapter_test.exs

```diff
@@ -0,0 +1,158 @@
+defmodule Phoenix.Integration.CodeGeneration.AppWithMSSQLAdapterTest do
+  use Phoenix.Integration.CodeGeneratorCase, async: true
+
+  describe "new with defaults" do
+    test "has no compilation or formatter warnings" do
+      with_installer_tmp("app_with_mssql_adapter", fn tmp_dir ->
```

Comment:
I think we can skip this one for mssql and mysql. This will by definition be tested when we test the generators later.